### ISSUE TITLE
CVE-2023-23397 check not sufficient - FP

### ISSUE THREAD

**Issue by LuemmelSec** [BC-SECURITY/Moriarty#19]
The script says that my system is prone to CVE-2023-23397.
Going after the debug log, it says so because it validated that the UNC path is in the wav file!?
If that is true, this is not a proof for being vulnerable. For that you would need to spin up something like responder to which the system that received the mail and opened the appointment actually would connect and auth to.
Testing this only locally seems to make no sense from my current perspective.
Also my system is up-to-date with all patches applied, so anyways should not be vulnerable.

![image](https://github.com/BC-SECURITY/Moriarty/assets/58529760/da318c52-4457-4300-816f-de1c43966a2e)

**Comment by Cx01N** [comment]
Yeah its hard when doing a lot of these scans and trying to keep them lightweight. Maybe it should be Vulnerable with a caveat that you should do a targettied check to confirm it.
